Decompiler/Options.cs

```diff
@@ -42,6 +42,18 @@ class Options
             HelpText = "Verify checksums and signatures.")]
         public bool VerifyVPKChecksums { get; set; }
 
+        [Option('d', "vpk_decompile", DefaultValue = false,
+            HelpText = "Decompile supported files")]
+        public bool Decompile { get; set; }
+
+        [Option('e', "vpk_extensions",
+            HelpText = "File extension(s) filter, example: vcss_c,vjs_c,vxml_c")]
+        public string ExtFilter { get; set; }
```

Comment:
From little google that I did it's not issue-free in this older version and the library looks abandoned. 
I'm sure there is a better way to use it, but for now my implementation seems to work ok. 
Also added fix for vpk_filepath to work with any directory separator